### ISSUE TITLE
Dev Tools Minimize Issue - Windows only

### ISSUE THREAD

**Issue by VishwasShashidhar** [electron/electron#11509]
* Electron version: 1.8.2 beta 3
* Operating system: Windows 10

### Expected behavior

When we toggle dev tools & pop out the dev tools window, minimize it, toggle the dev tools (to essentially close it), and toggle the dev tools again (to open it again), the dev tools window should be visible as expected.

### Actual behavior

When we minimize the dev tools window from a pop out, close it by toggling it again, and open it by toggling it, the dev tools window doesn't show anything, it's just blank. We need to restart the app to get the dev tools working again.

### How to reproduce

```
git clone https://github.com/electron/electron-api-demos
cd electron-api-demos
npm i electron@1.8.2-beta.3
npm i
npm start
```

Please see the below gif to better understand the context.

![is_0099111](https://user-images.githubusercontent.com/5968790/34297500-7f779f8c-e73e-11e7-9b62-9eee4b823da0.gif)

**Comment by anurocha** [comment]
I checked with Electron code.
seems this has been implemented on purposed as in https://github.com/electron/electron/blob/master/atom/browser/api/atom_api_web_contents.cc#L1237

ToggleDevTools API is to show/hide the dev window.
if you minimized the first window then you toggle again, it will close the dev window in minimized state.
finally, when you toggle it again, it will restore dev window in minimized state.
